### ISSUE TITLE
Deprecation Warning

### ISSUE THREAD

**Issue by breim** [joelhawksley/actionview-component-demo#29]
DEPRECATION WARNING: passing component instances to `render` has been deprecated and will be removed in v2.0.0. Use `render MyComponent, foo: :bar`

In view:
```
<%= render CardProfile::CardProfileComponent, influencer: @influencer, metadata: false do %>
   <h1>Super title</h1>
<% end %>

```

Depreciation is telling me to change **render** method to **render**. You might show a complete example?
Thx

**Comment by joelhawksley** [comment]
Good idea! Might you mind filing this in https://github.com/github/actionview-component/issues?

**Comment by 25161** [comment]
> CẢNH BÁO KHAI THÁC: chuyển các phiên bản thành phần `render`đã bị phản đối và sẽ bị xóa trong v2.0.0. Sử dụng`render MyComponent, foo: :bar`
> 
> Theo quan điểm:
> 
> ```
> <%= render CardProfile::CardProfileComponent, influencer: @influencer, metadata: false do %>
>    <h1>Super title</h1>
> <% end %>
> ```
> 
> Khấu hao được biết cần thay đổi **làm cho** phương pháp để **làm** . Bạn có thể hiển thị một ví dụ đầy đủ?
> Cám ơn
